spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalTaskLauncher.java

```diff
@@ -205,7 +204,8 @@ private Path createLogDir(AppDeploymentRequest request) throws IOException {
 							request.getDefinition().getName());
 		}
 
-		String qualifiedName = Long.toString(System.currentTimeMillis());
+		Instant now = Instant.now();
+		String qualifiedName = Long.toString(now.toEpochMilli() + now.getNano());
```

Comment:
Btw, this will not actually work as it has similar precision issue:
```
    	long t1 = System.currentTimeMillis();
    	long t2 = System.currentTimeMillis();
    	long t3 = System.nanoTime();
    	long t4 = System.nanoTime();

    	System.out.println(Long.toString(t1));
    	System.out.println(Long.toString(t2));
    	System.out.println(Long.toString(t3));
    	System.out.println(Long.toString(t4));

    	Instant now1 = Instant.now();
    	Instant now2 = Instant.now();
    	long t5 = now1.toEpochMilli() + now1.getNano();
    	long t6 = now2.toEpochMilli() + now2.getNano();
    	System.out.println(Long.toString(t5));
    	System.out.println(Long.toString(t6));
```
which results:
```
1542298923798
1542298923798
1752380359022364
1752380359022472
1543097923799
1543097923799

```